src/main/java/be/appwise/core/networking/proxyman/Packages.kt

```diff
@@ -0,0 +1,96 @@
+package be.appwise.core.networking.proxyman
+
+import android.graphics.Bitmap
+import android.util.Base64
+import androidx.core.graphics.drawable.toBitmap
+import java.io.ByteArrayOutputStream
+import java.lang.Exception
+import java.nio.charset.StandardCharsets
+
+
+class ConnectionPackage() : Data {
+    val device: Message.Device = Message.Device.current
```

Comment:
I'm not sure if like the view of `Device` and `Project` as an inner class of `Message`...
I don't mind having them in the same file, but should they be inside off the `Message` class? 🤔

Reply:
I followed the setup of the Atllantis library. If we want to let them publish it the setups should be the same

Reply:
Hmm... is that something they explained like that, or is that just a thought you had?
It's like I said in another comment, things can always be improved upon, we don't have to follow their design/structure to the letter.

Reply:
Ah ok I read it wrong sorry! I already did this change locally but they need to be inner classes of the package class :)